hack/download-kubebuilder-local.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
```

Comment:
Should we also have a license header in this file? I think we can probably follow up with it in a later PR, along with adding some presubmits for verifying license headers - I don't think it's a huge issue now... cc @adamhosier

Reply:
Aren't licence headers redundant when we've got a top-level `LICENSE` file?

Reply:
i don't think we need headers.